src/ProductConstructionService/ProductConstructionService.Cli/Options/PCSStatusOptions.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using CommandLine;
+using Microsoft.Extensions.DependencyInjection;
+using ProductConstructionService.Client;
+
+namespace ProductConstructionService.Cli.Options;
+
+internal abstract class PCSStatusOptions : Options
+{
+    [Option("isCi", Required = false, HelpText = "Is running in CI")]
+    public required bool IsCi { get; init; } = false;
+    [Option("pcsUri", Required = false, HelpText = "Uri to PCS")]
```

Comment:
Can you mention in the helptext what is the default?

Reply:
added